### ISSUE TITLE
A confusion in hubconf.py

### ISSUE THREAD

**Issue by coconut0713** [milesial/Pytorch-UNet#494]
- code:

> state_dict = torch.hub.load_state_dict_from_url(checkpoint, progress=True)
> if 'mask_values' in state_dict: state_dict.pop('mask_values')  
> net.load_state_dict(state_dict) 

Generally, ‘state_dict’ contains module names & parameters, but not mask values. What is the purpose of 'state_dict.pop('mask_values')  '?
